src/configValidator.js

```diff
@@ -39,15 +39,36 @@ function isLocalConfigValid(config) {
   return isValid(missingConfigFields);
 }
 
+function isMobileConfigValid(config) {
+  let isMobileConfigCorrect = true;
+
+  config.scenarios.forEach(scenario => {
```

Comment:
You could use 'array.some' here. https://www.w3schools.com/jsref/jsref_some.asp.
It would save the need for 'isMobileConfigCorrect' as you can just return the output.